integration/basic_test.go

```diff
@@ -31,6 +31,32 @@ func (s *RunSuite) TestHelloWorld(c *C) {
 	c.Assert(cn.Name, Equals, "/"+name)
 }
 
+func (s *RunSuite) TestInterpolation(c *C) {
```

Comment:
Can you add another test or include it in this one that validates that interpolation happens for [extended templates](docs.docker.com/compose/yml/#extends) meaning that the parent and child templates both have variables in them.